LEGO1/lego/legoomni/src/paths/legoanimactor.cpp

```diff
@@ -1,27 +1,200 @@
 #include "legoanimactor.h"
 
+#include "define.h"
+#include "legoanimpresenter.h"
+#include "legoworld.h"
+#include "misc.h"
+#include "mxutilities.h"
+
 DECOMP_SIZE_ASSERT(LegoAnimActor, 0x174)
+DECOMP_SIZE_ASSERT(LegoAnimActorStruct, 0x20)
+
+// FUNCTION: LEGO1 0x1001bf80
+LegoAnimActorStruct::LegoAnimActorStruct(float p_float, LegoAnim* p_animTreePtr, LegoROI** p_roiMap, MxU32 p_numROIs)
+{
+	m_unk0x00 = p_float;
```

Comment:
Generally parameter names should be named the same way as the members, in this case `p_unk0x00`